tests/components/airos/test_init.py

```diff
@@ -0,0 +1,42 @@
+"""Test for airOS integration setup."""
+
+from __future__ import annotations
+
+from unittest.mock import MagicMock
+
+from homeassistant.components.airos.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+MOCK_CONFIG_V1 = {
+    CONF_HOST: "1.1.1.1",
+    CONF_USERNAME: "ubnt",
+    CONF_PASSWORD: "test-password",
+}
+
+
+async def test_load_unload_entry(
```

Comment:
This test doesn't actually test the added feature at all.?

Reply:
You could test that the mocked object get's called with the correct value.

Reply:
Yeah that would make sense - (re)added both actually